src/components/calltree/CallTree.js

```diff
@@ -91,79 +93,42 @@ class CallTreeImpl extends PureComponent<Props> {
       switch (weightType) {
         case 'tracing-ms':
           return [
-            { propName: 'totalPercent', title: '' },
+            { propName: 'totalPercent', titleL10nId: '' },
```

Comment:
I guess I could also make them either null or undefined but didn't want to change the current way.